### ISSUE TITLE
追加事前学習の記事等

### ISSUE THREAD

**Issue by takumi7110** [takumi7110/pretraining#1]
記事
・法律系のdataで
専門知識を持つデータで事前学習することの重要性とは？ ~ 法律文書の分類に対するBERTの応用 ~
https://ai-scholar.tech/articles/natural-language-processing/legal-document-classification-bert
論文:https://aclanthology.org/2021.nllp-1.22/
dataは、・ECHR Violation Dataset (Multi Label) 、・Overruling Task Dataset  

【論文解説】Don’t Stop Pretraining – 追加で事前学習を行う効果について
https://data-analytics.fun/2020/10/12/understanding-better-pretraining/

legal bert
https://huggingface.co/casehold

・金融系のdataで
事前学習と追加事前学習による金融言語モデルの構築と検証
Construction and Validation of a Pre-Training and Additional Pre-Training Financial Language Model
https://www.jstage.jst.go.jp/article/jsaisigtwo/2022/FIN-028/2022_132/_pdf
dataはhttps://huggingface.co/izumi-lab等

FinGPT
https://github.com/AI4Finance-Foundation/FinGPT/blob/master/FinGPT_Training_LoRA_with_ChatGLM2_6B_for_Beginners.ipynb

DataSet
・中国の生物系のcorpus 
https://tianchi.aliyun.com/dataset/95414
・
